roles/load_aware/load_aware_scale_test/files/load-aware-ns.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: load-aware
+  name: coreutils
   labels:
-    name: load-aware
+    name: coreutils
```

Comment:
I'm not sure why you're changing the namespace, I preferred the previous `load-aware` name 🤔 
a better solution would be to make it configurable in the config file, and here use the value from parameter

Reply:
> here

here and everywhere in the role :)

Reply:
The `coreutils` namespace was originally created as a place to store the intermediate imagestreams and buildconfigs to be used in both of the tests. The `load-aware` namespace was used used for the workload pods. This caused issue since the workload pods don't have access by default to the imagestreams in the other namespace. I put them all in the same namespace, it just so happen to be the coreutils one.

I can make this configurable though.